lib/galaxy/webapps/base/controller.py

```diff
@@ -1577,3 +1577,15 @@ def sort_by_attr(seq, attr):
     intermed = [(getattr(v, attr), i, v) for i, v in enumerate(seq)]
     intermed.sort()
     return [_[-1] for _ in intermed]
+
+
+__all__ = (
+    "HTTPBadRequest",
+    "SharableMixin",
+    "sort_by_attr",
+    "url_for",
+    "UsesExtendedMetadataMixin",
+    "UsesFormDefinitionsMixin",
+    "UsesTagsMixin",
+    "web",
```

Comment:
Re-exported, imported from here in `lib/galaxy/webapps/galaxy/controllers/error.py` , `lib/galaxy/webapps/galaxy/controllers/forms.py` , `lib/galaxy/webapps/reports/controllers/history.py` , `lib/galaxy/webapps/reports/controllers/home.py` , `lib/galaxy/webapps/reports/controllers/jobs.py` , `lib/galaxy/webapps/reports/controllers/root.py`, `lib/galaxy/webapps/reports/controllers/system.py` , `lib/galaxy/webapps/reports/controllers/tools.py` , `lib/galaxy/webapps/reports/controllers/users.py` , `lib/galaxy/webapps/reports/controllers/workflows.py`